### DISCUSSION TITLE
Add `d2` to From text options

### DISCUSSION THREAD

**Discussion by alixander** [jgraph/drawio#4274] [General]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

Would the drawio team accept a PR to include d2 as a From Text option next to Mermaid and PlantUML? https://github.com/terrastruct/d2

~~If so, I'm happy to submit PR for implementation. (although I could not find this file: https://github.com/jgraph/drawio/blob/dev/src/main/webapp/js/diagramly/Devel.js#L294 . is the comment just outdated? Can you point me to where the equivalent would be found?)~~
edit: oops, forgot it's not open-contribution. I guess question then is whether drawio would support this and if there's anything I can do to help

**Comment by davidjgraph** [comment]
Thanks for the suggestion. We can leave this open, but it's not something we'd look into in the next 12 months. We're focusing on Mermaid as the primary supported text input format for the time being.
